website/docs/api/large-language-models.mdx

```diff
@@ -236,6 +236,57 @@ objects. This depends on the return type of the [model](#models).
 | `responses` | The generated prompts. ~~Iterable[Any]~~   |
 | **RETURNS** | The annotated documents. ~~Iterable[Doc]~~ |
 
+### Translation {id="translation"}
+
+The translation task translates texts from a defined or inferred source to a
+defined target language.
+
+#### spacy.Translation.v1 {id="translation-v1"}
+
+`spacy.Translation.v1` supports both zero-shot and few-shot prompting.
+
+> #### Example config
+>
+> ```ini
+> [components.llm.task]
+> @llm_tasks = "spacy.Translation.v1"
+> examples = null
+> target_lang = "Spanish"
+> ```
+
+| Argument                    | Description                                                                                                                                                                                   |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                  | Custom prompt template to send to LLM model. Defaults to [summarization.v1.jinja](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/tasks/templates/summarization.v1.jinja). ~~str~~ |
+| `examples`                  | Optional function that generates examples for few-shot learning. Defaults to `None`. ~~Optional[Callable[[], Iterable[Any]]]~~                                                                |
+| `parse_responses` (NEW)     | Callable for parsing LLM responses for this task. Defaults to the internal parsing method for this task. ~~Optional[TaskResponseParser[SummarizationTask]]~~                                  |
+| `prompt_example_type` (NEW) | Type to use for fewshot examples. Defaults to `SummarizationExample`. ~~Optional[Type[FewshotExample]]~~                                                                                      |
```

Comment:
Update c/p error referring to "summarization" twice more 😉

Reply:
:upside_down_face: Fixed in https://github.com/explosion/spaCy/pull/13183/commits/6265e0c52886525df21b39f8f762bfed91e7732d.